content/docs/0.16.0-next.x/reference/cli/commands/keptn_update_project.md

```diff
@@ -0,0 +1,79 @@
+---
+date: "2022-05-31T06:17:10Z"
+title: "keptn update project"
+slug: keptn_update_project
+---
+## keptn update project
+
+Updates an existing Keptn project
+
+### Synopsis
+
+Updates an existing Keptn project with the provided name. 
+
+Updating a shipyard file is not possible.
+
+By executing the update project command, Keptn will add the provided upstream repository to the existing internal Git repository that is used to maintain all project-related resources. 
+To upstream this internal Git repository to a remote repository and the remote URL (*--git-remote-url*) are required
```

Comment:
```suggestion
To upstream this internal Git repository to a remote repository, the name of the remote repository and the remote URL (*--git-remote-url*) are required.
```
Is this what is meant?  Something is screwy with this sentence in its current form.  Also, we don't seem to have a Synopsis for this command.